src/shared/dissect-image.c

```diff
@@ -917,7 +939,10 @@ static int dissect_image(
 
                 /* If flags permit this, also allow using non-partitioned single-filesystem images */
 
-                (void) sym_blkid_probe_lookup_value(b, "USAGE", &usage, NULL);
+                if (!(flags & DISSECT_IMAGE_NO_BLKID))
+                        (void) sym_blkid_probe_lookup_value(b, "USAGE", &usage, NULL);
+                else
+                        usage = "filesystem";
```

Comment:
here too we should look at the policy, and determine the right thing: i.e. if the policy says "no encryption" then it should use "filesystem". if the policy however says: "must be encryption", then default to "crypto" here. and if it leaves this unspeicifed we should go the blkid route.